### ISSUE TITLE
Update db call to to receive email strings back from the db in lower case form

### ISSUE THREAD

**Issue by trillium** [hackforla/VRMS#1480]
### Dependency
- [x] Complete after #1551 

### Overview
We need to receive email addresses as lowercase from the db so that duplicates of different casing (jeff@exmaple.com, JEFF@example.com) aren't able to be created.

### Action Items
- [ ] Update the db so that all emails are already lowercase
(this is the preferred solution, but we will need to handle duplicate addresses which increases complexity)
- [ ] Merge PR into feature branch: [`correct-db-casing` ](https://github.com/hackforla/VRMS/tree/correct-db-casing)

### Resources
 - [MongoDB Docs](https://www.mongodb.com/docs/manual/)
 - This issue is part of the epic: #1456

**Comment by trillium** [comment]
We're in luck here, the production DB only has two duplicate emails eg `jeff@example, Jeff@example` and ~40 emails that have upper case letters in them. Enough that a manual edit isn't a huge burden.

That means we don't need to update the DB call to return an email in lower case format. Win!

**Comment by bkmorgan3** [comment]
@Spiteless  There is a { lowercase: true } value that can be added at the db layer to avoid all these manipulations on the FE.

**Comment by JackHaeg** [comment]
This issue is being addressed by #1551. Once that ticket is finished, we can address this issue.

**Comment by dannyprikaz** [comment]
We have determined that enforcing that emails are saved as lowercase is a better solution than returning them as lowercase regardless of how they are stored. A new issue has been created (#1978), and this one will be closed.
